classes/class-admin.php

```diff
@@ -25,6 +25,11 @@ class Admin {
 	public $live_update;
 
 	/**
+	 * @var Export
+	 */
+	public $export;
```

Comment:
Right now the exporter specifically takes Stream Records from a List Table and Exports. So I would probably place it with `List_Table`.